command/job_inspect.go

```diff
@@ -206,3 +241,58 @@ func getJob(client *api.Client, namespace, jobID string, version *uint64) (*api.
 
 	return nil, fmt.Errorf("job %q with version %d couldn't be found", jobID, *version)
 }
+
+// getJob retrieves the job optionally at a particular version.
+func getJobHCL(client *api.Client, namespace, jobID string, version *uint64) (*api.JobSubmission, error) {
+	var q *api.QueryOptions
+	if namespace != "" {
+		q = &api.QueryOptions{Namespace: namespace}
+	}
+	v := uint64(0)
+	if version != nil {
+		v = *version
+	} else {
+		job, _, err := client.Jobs().Info(jobID, q)
+		if err != nil {
+			return nil, err
+		}
+		v = *job.Version
+	}
+	submission, _, err := client.Jobs().Submission(jobID, int(v), q)
+
+	if err != nil {
+		return nil, fmt.Errorf("job %q with version %d couldn't be found", jobID, version)
+	}
+	return submission, err
+}
+
+func getWithVarsOutput(namespace, jobID string, uiVars string, varsMap map[string]string) string {
+	runArgs := []string{}
+	if namespace != "" {
+		runArgs = append(runArgs, "-namespace")
+		runArgs = append(runArgs, namespace)
+	}
+
+	if len(varsMap) != 0 {
```

Comment:
the length check is redundant - the `range` will do nothing if the map is empty anyway